WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -689,6 +709,32 @@ public void onPostChanged(OnPostChanged event) {
     public void onPostUploaded(OnPostUploaded event) {
         if (isAdded() && event.post.getLocalSiteId() == mSite.getId()) {
             loadPosts(LoadMode.FORCED);
+
+            final PostModel post = event.post;
+            boolean isDraft = post != null && PostStatus.fromPost(post) == PostStatus.DRAFT;
+            if (isDraft) {
+                if (PostUtils.isPublishable(post)) {
```

Comment:
Oh you're right. Sorry, I was thinking of the logic in `handleEditPostResult()` which is partially similar and got confused, but of course just like you said: something will necessarily pass the `isPublishable()` condition if we've already gotten the `OnPostUploaded` signal. Sorry about that.
BTW the `isPublishable` name is fine for me.